src/worker.js

```diff
@@ -39,8 +39,13 @@ if (ENVIRONMENT_IS_NODE) {
     require,
     Module,
     location: {
-      // __filename is undefined in ES6 modules
+      // __filename is undefined in ES6 modules, and import.meta.url only in ES6
+      // modules.
+#if EXPORT_ES6
       href: typeof __filename !== 'undefined' ? __filename : import.meta.url
+#else
+      href: typeof __filename
```

Comment:
What happened here?

Reply:
This isn't necessary, I think, but we only need this if EXPORT_ES6, so it saves some code size? Maybe not worth it though.

Reply:
Oh, this is just a typeof now... yes, that was a mistake.

But I see meanwhile this is needed. `nodejs` will error on just seeing `import.meta` outside of an ES6 module, it doesn't even need to reach it to execute it.

Reply:
(The if-else is needed, I think. Thanks for catching the `typeof`!)

Reply:
Yeah my concern was about `typeof`, `if-else` itself is fine.